.changeset/entries/f7bb961b8872d11fc00ebe6d43c2e5a6fae4f1fb0e63eb9f8b2d3db360260b97.yaml

```diff
@@ -0,0 +1,6 @@
+type: fix
+module: other
+pull_request: 1069
+description: Register missing Wasm snapshotter extension
+backward_compatible: true
+date: 2023-01-17T16:57:07.714192798Z
```

Comment:
Can you remove it then add it to CHANGELOG.md by changeset?

Reply:
Thanks for pointing out, I've missed it! Done now :pray: